### PR TITLE
Create new npm package

### PR DESCRIPTION
**Pull request by Shwetajain148**
As we have to create new npm package. Below are the changes that are done in this PR.

-update package version
-update author details
-update README.md reference

Created Package link: [loggly-winston-bulk](https://www.npmjs.com/package/loggly-winston-bulk).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Once you have valid configuration and credentials you can run tests with [npm][7
   npm test
 ```
 
-#### Author: [Charlie Robbins](http://blog.nodejitsu.com)
+#### Author: [Shweta Jain](https://www.loggly.com/)
```
Comment (mchaudhary) [inline README.md:85]: Should we change name ? @mostlyjason  thoughts ?

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "winston-loggly-bulk",
-  "version": "1.3.1",
+  "name": "loggly-winston-bulk",
+  "version": "1.0.1",
   "description": "A Loggly transport for winston",
-  "author": "Charlie Robbins <charlie.robbins@gmail.com>",
+  "author": "Shweta Jain <sjain@loggly.com>",
```
Comment (mchaudhary) [inline package.json:5]: same here?

File: README.md (hunk 1 above)
Comment (Shwetajain148) [reply README.md:85]: @mostlyjason Please let me know your views on this.

Comment (mostlyjason) [reply README.md:85]: Sure we can update it or you can add a 2nd line

Comment (Shwetajain148) [reply README.md:85]: @mchaudhary @mostlyjason I have made changes in the README.md file as per our discussion on author name. Please review..

**Pull request merged**